mlir/test/Target/Cpp/lvalue.mlir

```diff
@@ -0,0 +1,22 @@
+// RUN: mlir-translate -mlir-to-cpp %s | FileCheck %s
+
+// CHECK: int32_t lvalue_variables(
+emitc.func @lvalue_variables(%v1: i32, %v2: i32) -> i32 {
```

Comment:
I don't quite get this test - I would expect more check lines?

Reply:
Unfinished. I wanted to get the conversion working first before putting much effort into tidying up.